perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -454,6 +454,23 @@ sub inspect : Private {
                 $redirect_uri = $c->uri_for_action('my/planned');
             }
 
+            # Sign up to alerts by default
+            my $options = {
```

Comment:
As this code very much overlaps with that in `Report/Update.pm`, could this be factored out? Presumably as a method on either Problem or User, can't decide which :)